### PR TITLE
org.apache.jmeter

### PR DESCRIPTION
**Pull request by prodrigestivill**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** [upstream issue][upstream-issue]
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission
[upstream-issue]: https://bz.apache.org/bugzilla/show_bug.cgi?id=66258

### Permissions

I added `filesystem=xdg-documents` in order to be able to save the project files, not sure what is the expected permissions to be set by default for such applications.

### Patches or hacks

A wrapper script has been added to configure the configuration files and plugin paths following the XDG structure.

### CODE REVIEW

File: org.apache.jmeter.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <p>Apache JMeter is a desktop application to design and run load test and measurements.</p>
+  </description>
+  <url type="homepage">https://jmeter.apache.org/</url>
+  <url type="bugtracker">https://jmeter.apache.org/issues.html</url>
+  <url type="faq">https://cwiki.apache.org/confluence/display/JMETER/JMeterFAQ</url>
+  <url type="help">https://jmeter.apache.org/usermanual/index.html</url>
+  <url type="donation">https://www.apache.org/foundation/contributing.html</url>
+  <url type="contact">http://jmeter.apache.org/mail2.html</url>
+  <!-- TODO
+  <screenshots>
```
Comment (hfiguiere) [inline org.apache.jmeter.metainfo.xml:19]: screenshots are required. the build will fail without

File: org.apache.jmeter.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --env=PATH=/app/bin:/app/jre/bin:/usr/bin
+  - --filesystem=xdg-documents
+
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands: [/usr/lib/sdk/openjdk17/install.sh]
```
Comment (hfiguiere) [inline org.apache.jmeter.yaml:21]:
```suggestion
    build-commands: 
      - /usr/lib/sdk/openjdk17/install.sh
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, build org.apache.jmeter

File: org.apache.jmeter.metainfo.xml (hunk 1 above)
Comment (prodrigestivill) [reply org.apache.jmeter.metainfo.xml:20]: Added screenshots from the official README file

**Comment by prodrigestivill** [comment]
bot, build org.apache.jmeter

**Pull request closed without merging**